### ISSUE TITLE
Export folder World Downloader

### ISSUE THREAD

**Issue by Ulex0000** [Pokechu22/WorldDownloader#170]
Hi guys,

I've just downloaded and launched world downloader and downloaded a world which I can access by using the minecraft app created by world downloader. When I join it, it is is perfectly saved. Here we come to the problem... when I go to MultiMC, I open the folder where the world is saved (I'm sure is that because has the same name of the world in singleplayer) and drag it into my .minecraft --> saved. Then, when I log with my normal account (not game launched by MultiMC), I find the world in singleplayer with the correct icon and name, but when I join it it doesn't load the world... It apperars like a normal void world... I've also tried to /tp myselfe to the right coordinates, but I still get the void... so, in conclusion, the downloaded world doesn't load.

Can you help me????

P.S. after having taken away the world folder from the "saved" of the launcher, if I reput the folder in it, it doesn't load it too...

**Comment by Pokechu22** [comment]
To make sure I understand, the world loads correctly when you open it in singleplayer immediately after downloading it when using multimc, but then when you move it to your normal minecraft install it stops working?

**Comment by Ulex0000** [comment]
Yes, you got that!

**Comment by Pokechu22** [comment]
By default, WDL sets the terrain generator to only generate void (to avoid causing weird world boundaries when generating new chunks).  If the downloaded world became void, something probably caused it to regenerate itself.  The most likely thing I can think of is you saved it in one Minecraft version, but then tried to load it in an earlier version, which doesn't know how to load the data and ends up discarding it.

There should be a zipped copy of the world you downloaded in the `backups` folder in your multimc install (it's in the same folder as the `saves` folder), so you shouldn't need to redownload everything if the original one was corrupted by using the wrong MC version.  (You should also be able to extract it into the multimc saves folder and test it there.)

**Comment by Ulex0000** [comment]
It might be because I installed word downloader for 1.15.2, then I uploaded it on my server and tried to import it with multiverse... the problem is that the server is in 1.14.... I may install multimc for 1.14 and try again....

In any case, when I dragged the folder of the downloaded world into the desktop, I tried to put it again in the saves of multimc, but the result was that, even if I loaded it with the save versione that I used to download it, it doesn’t load there too.... is strange... 

To recap, when I download it in 1.15.2 and drag it away, when I reload it on multimc (so, yes I take it away from multimc and then immediately put the folder again in it) it doesn’t load anymore.

Hope to be enough clear... sorry but is almost bed time😅😅😅

**Comment by Ulex0000** [comment]
I don’t believe that it’s a problem correlated to the version of Minecraft... never happened that a world doesn’t load at at when using the same blocks, at least...

**Comment by Pokechu22** [comment]
If you load a world from 1.13 in 1.12 (or earlier), then everything in it will be lost (due to different save formats from the flattening).  I don't remember if the same happens with 1.15 to 1.14, but it might (I think there were light format changes of some sort?).  Mojang doesn't support loading newer worlds in older versions (they only support upgrading older worlds to the newer formats).  Loading the world and then exiting it will cause the regenerated (and now empty) chunks to be saved, so if you move it back to a newer version you will get chunks still.

If you plan on using the world in 1.14.4, it's a lot easier to use the 1.14.4 version of WDL.

**Comment by Ulex0000** [comment]
Ok fine. I’ll try and let you know! 
Thank you for the support given!

**Comment by Ulex0000** [comment]
Why does it tell me this? How can I fix it?

![Uploading CCA5AA8B-B526-4438-AA12-EEB3593F41C8.jpeg…]()

**Comment by Ulex0000** [comment]
Dam... why does the MultiMC launcher shows ❌Session.... I can’t join servers so...

**Comment by Ulex0000** [comment]
So, if I load it as default world on my 1.14 server, I get the void... if I load it with multiverse-core, it’s all fine.... not normal... 
(When i put the folder intro the server data, I renamed the folder as “word”).

This is strange btw...

**Comment by Pokechu22** [comment]
Did the world saved in 1.14 load in singleplayer 1.14?  Also, did you have multiworld enabled when you were downloading?  (If you have multiworld enabled, WDL asks you which world you're in when you start downloading.)

Note that if your server is modded, the world layout is slightly different that of vanilla, specifically with regards to other dimensions (instead of having other dimensions in the `DIM-1` and `DIM1` subfolders of the main world, there are separate `world_nether` and `world_the_end` folders).  This is the case even without multiverse.  To my understanding, it should automatically be converted to the split format, though.

**Comment by Ulex0000** [comment]
I disable the plug in for that occasion, but nothing... when I name the downloaded world as “world” and delete the original one, it doesn’t load the game... while, if I Import it with multiverse, it loads perfectly... I’m just going to set that world as the spawn one, but this isn’t what I expected....

**Comment by Ulex0000** [comment]
Fine! It works! Thank you for your support!

**Comment by Ulex0000** [comment]
I just had to log with the same version which I used to download the game!
